tardis/conftest.py

```diff
@@ -110,7 +92,40 @@ def included_he_atomic_data(test_data_path):
     return AtomData.from_hdf5(atomic_db_fname)
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
 def tardis_config_verysimple():
     return yaml_load_config_file(
         'tardis/io/tests/data/tardis_configv1_verysimple.yml')
+
+
+@pytest.fixture(scope="session")
+def _atom_data(request):
+    atom_data_name = 'kurucz_cd23_chianti_H_He.h5'
+    # Download and cache the zip file of atomic data.
+    atom_data_cache = download_file(
+        'http://www.mpa-garching.mpg.de/~michi/tardis/data/kurucz_cd23_chianti_H_He.zip',
+        cache=True
+    )
+
+    # Obtained file is a zip file, hence unzipped inside a tempdir.
+    atom_data_zipfile = zipfile.ZipFile(atom_data_cache)
+    atom_data_extract_tempdir = tempfile.mkdtemp()
+    atom_data_zipfile.extract(atom_data_name, path=atom_data_extract_tempdir)
+
+    atom_data = AtomData.from_hdf5(
+        os.path.join(atom_data_extract_tempdir, atom_data_name)
+    )
+
+    def fin():
+        # Delete the tempdir as no longer required.
+        shutil.rmtree(atom_data_extract_tempdir)
+    request.addfinalizer(fin)
+    return atom_data
+
+
+@pytest.fixture(scope="class")
+def atom_data(_atom_data):
+    if _atom_data.md5 != '21095dd25faa1683f4c90c911a00c3f8':
+        pytest.skip('Need default Kurucz atomic dataset '
+                    '(md5="21095dd25faa1683f4c90c911a00c3f8"')
+    return copy.deepcopy(_atom_data)
```

Comment:
As `copy.deepcopy` doesn't seem to work here we might as well get rid of `atom_data` and rename `_atom_data` to `atom_data`